tests/admin_inlines/tests.py

```diff
@@ -750,9 +750,10 @@ def test_delete_stackeds(self):
         add_button.click()
 
         self.assertEqual(rows_length(), 5, msg="sanity check")
-        for delete_link in self.selenium.find_elements_by_css_selector(
-                '%s .inline-deletelink' % inline_id):
-            delete_link.click()
+        self.selenium.find_element_by_css_selector(
```

Comment:
This can be 1 line (we favor lines up to 119 characters when it helps readability)